### PR TITLE
dev/EC-27 open close chevron

### PR DESCRIPTION
**Pull request by IgorKulishov**
1) changed chevron svg image to arrow (matching with mock arrow in ticket)
2) added space between order name and arrow
3) created indexed array for each accordion to keep state of open and closed position
4) reused existing function to pass index and change state of accordion in indexed array
5) used existing css class and changed styles to
- define height of svg arrow 
- used existing animation to rotate 90 deg (for opening)
6) added css class for closed block 
- define height of svg arrow 
- used existing animation to rotate 0 deg (for closing)

### CODE REVIEW

File: src/app/cart/components/placed-orders/placed-orders.component.html [hunk 1]
```diff
@@ -49,8 +49,16 @@
         <accordion [isAnimated]="true">
           <accordion-group>
             <button id="modalBtn" class="btn btn-link" accordion-heading type="button" (click)='changeChevronDirection(i)'>
-              {{placedOrdersDetails?.updatedAt | date}}
-              <img [attr.id]="'chevCaretRight'+i" src="../../../../assets/svg/caret-right-fill.svg" alt="Right caret">
+
+
```
Comment (g1er) [inline src/app/cart/components/placed-orders/placed-orders.component.html]: Pls delete empty lines

File: src/app/products/components/create-product/add-details/add-details.component.spec.ts [hunk 2]
```diff
@@ -2,11 +2,11 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { StoreModule } from '@ngrx/store';
 import {ReactiveFormsModule, FormsModule} from '@angular/forms';
 import { Router, ActivatedRoute } from '@angular/router';
-
+import { provideMockStore, MockStore } from '@ngrx/store/testing';
```
Comment (g1er) [inline src/app/products/components/create-product/add-details/add-details.component.spec.ts]: Not used imports. Pls delete

Comment (IgorKulishov) [reply src/app/products/components/create-product/add-details/add-details.component.spec.ts]: Thank you
Deleted

File: src/app/cart/components/placed-orders/placed-orders.component.html (hunk 1 above)
Comment (IgorKulishov) [reply src/app/cart/components/placed-orders/placed-orders.component.html]: Deleted

**Review by g1er — APPROVED** [review]

**Pull request merged**